client/my-sites/post-share/index.jsx

```diff
@@ -357,38 +338,56 @@ class PostShare extends Component {
 			'has-connections': this.hasConnections()
 		} );
 
+		const {
+			connections,
+			failure,
+			postId,
+			requesting,
+			siteId,
+			siteSlug,
+			success,
+			translate,
+		} = this.props;
+
+		if ( ! siteId || ! postId ) {
+			return null;
+		}
+
 		return (
 			<div className="post-share">
-				{ this.props.requesting &&
+				{ requesting &&
 					<Notice status="is-warning" showDismiss={ false }>
-							{ this.props.translate( 'Sharing…' ) }
+							{ translate( 'Sharing…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Sharing...' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/post-share-state-connection)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).